codefellowship/codefellowship/src/main/java/com/codefellowship/controllers/AccountController.java

```diff
@@ -116,8 +118,16 @@ public String myProfile(@RequestParam Long id,
         return "userdetails";
```

Comment:
on line 116, you need to assign a name to the attribute. 
model.addAttribute("user", user.get());

Reply:
Line 111 should also have an @PathVariable annotation instead.